README.md

```diff
@@ -47,9 +47,7 @@
 
 ## Recommended Installation
 
-The package `prymer` requires installation of [interactive `bwa`](https://github.com/fulcrumgenomics/bwa-aln-interactive).
-
-To satisfy these requirements, it is recommended to install using [bioconda](https://bioconda.github.io/):
+It is recommended to install using [bioconda](https://bioconda.github.io/):
```

Comment:
Once this PR is merged I hope to make a small PR to remove this statement, and fix up CONTRIBUTING to make it clear that you can develop `prymer` with _just_ poetry available and managing the virtual envs.